src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -118,11 +118,11 @@ public void crossCheckMaximumVitalsValidity() {
 				hasItem("Maximum: 230"));
 		patientCaptureVitalsPage.clearPatientPulse();
 		patientCaptureVitalsPage.setPulseField("78");
-		patientCaptureVitalsPage.setRespiratoryField("1004");
+		patientCaptureVitalsPage.setRespiratoryField("106");
```

Comment:
This repository field is the caus of the failures. can you please change this to maximum value less than 99. this implies you are still testing on demo server which is not up to date with a server.

Reply:
> can you please change this to maximum value less than 99

The method [crossCheckMaximumVitalsValidity()](https://github.com/openmrs/openmrs-contrib-qaframework/blob/dc7ff801d2126546a8b58eb56449f9986a5c5611/src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java#L100) is asserting that a vital can be saved in the database if its above the maximum value of that given parameter.  

For this test to validate the respiratory rate value, the first entered value must be above the maximum value `patientCaptureVitalsPage.setRespiratoryField("106");` and then it validates the value from at line code [122](https://github.com/openmrs/openmrs-contrib-qaframework/blob/dc7ff801d2126546a8b58eb56449f9986a5c5611/src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java#L122) and then the test sets the value that is within the range at code line [125](https://github.com/openmrs/openmrs-contrib-qaframework/blob/dc7ff801d2126546a8b58eb56449f9986a5c5611/src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java#L125)